taxcalc/records.py

```diff
@@ -661,3 +656,30 @@ def read(self, data):
         self.e60100 = self.p60100
         self.e27860 = self.s27860
         self.SOIYR = np.repeat(2008, self.dim)
+
+    def mutate_imputations(self):
+        self._cmbtp_itemizer = imputation(self.e17500, self.e00100,
+            self.e18400, self.e18425, self.e62100, self.e00700,
+            self.e04470, self.e21040, self.e18500, self.e20800)
+
+
+@vectorize([float64(float64, float64, float64, float64, float64, float64,
+            float64, float64, float64, float64)])
+def imputation(e17500, e00100, e18400, e18425, e62100, e00700, e04470,
```

Comment:
ok great! @MattHJensen was hoping to get rid of specifying what type of function vectorize was using ie her preferred not to have: [float64(float64, float64, float64, float64, float64, float64, float64, float64, float64, float64)]. However, I could not get the code to run without it (despite updating numba)